evergreen/archive-evergreen.sh

```diff
@@ -18,12 +21,17 @@ cd -
 # working directory to "ARCHIVE_PATH" is a workaround.
 ARCHIVE_PATH=archive
 # Example archive file name: envoy-serverless-1.21.3-4-gdbdcfa34cf.rhel7.amd64.tar.gz
-ARCHIVE_DIR=envoy-serverless-${VERSION}.rhel7.amd64
+ARCHIVE_DIR=envoy-serverless-${VERSION}.${DISTRO}.${ARCH}
 mkdir -p ${ARCHIVE_PATH}/${ARCHIVE_DIR}
 
+if [ $ARCH == "aarch64" ]; then
+	OUTPUT_DIR="aarch64"
+else
+	OUTPUT_DIR="k8"
```

Comment:
This is weird.  This is just how the Envoy build is configured I guess?

Reply:
Yes I've noticed this as well. We use `k8` for Linux `x86_64` (https://github.com/envoyproxy/envoy/blob/93b76c70ae0153452109934818c805b63d8f1221/bazel/BUILD#L503-L506)